src/irr/asset/CPLYMeshFileLoader.cpp

```diff
@@ -655,7 +683,7 @@ bool CPLYMeshFileLoader::genVertBuffersForMBuffer(asset::ICPUMeshBuffer* _mbuf,
 		primitiveAssemblyParams.primitiveType = E_PRIMITIVE_TOPOLOGY::EPT_POINT_LIST;
 
 	SRasterizationParams rastarizationParmas;
-	//rastarizationParmas.faceCullingMode = EFCM_NONE;
+	rastarizationParmas.faceCullingMode = EFCM_NONE;
```

Comment:
why did you turn off culling?

Reply:
because we have tested it with Criss and didn't work without it

Reply:
that means that you have your polygons the wrong way around

Reply:
They are correct after the fix however